<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background:red;
}
#headerFrame
{
background: black;
border-radius: 30px;
}

#frame
{
background: red;
border-radius: 30px;
}
#labelHead{
color:white;
}

#label{
color:white;
}

#label_2{
color:white;
}

#lineEdit_ID{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_SuppName{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_supp_loc{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_supp_address{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_pquantity{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_psupplier{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#lineEdit_pprice{
border:none;
background:transparent;
color: white;
border-bottom: 2px solid white;
}

#pushButton_saveS{
border:2px solid white;
background:transparent;
color:white;
border-radius: 30px;
}

#pushButton_saveS:hover{
border:2px solid white;
background:white;
color:red;
}

#push_Rigister{
border:2px solid white;
background:transparent;
color:white;
border-radius: 30px;
}

#push_Rigister:hover{
border:2px solid white;
background:white;
color:red;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_supp_address">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>283</y>
     <width>421</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_supp_loc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>224</y>
     <width>421</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>576</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save new info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_register">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>61</y>
     <width>431</width>
     <height>38</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Change Supplier info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SuppName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>166</y>
     <width>421</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>421</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Aero</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTipDuration">
    <number>-5</number>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>Supplier ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
